### ISSUE TITLE
[Request] Lock object to Camera

### ISSUE THREAD

**Issue by degriz** [buddhi1980/mandelbulber2#709]
How about adding the ability to lock an objects position relative to the camera? So it moves when the camera moves to maintain its relative position.

**Comment by mclarekin** [comment]
yep, somewhere there is a "nice to have" issue, suggesting locking primitive group to camera

**Comment by degriz** [comment]
Lights to object would be fantastic, could have kinds of full making cone light and grills for that "Prison Window" effect

**Comment by mclarekin** [comment]
i would like to attach lights to my space rocket, instead of having to manipulate them separately in a spread sheet :)
